gradio/utils.py

```diff
@@ -130,6 +146,65 @@ def launch_analytics_thread(data: Dict[str, Any]) -> None:
     threading.Thread(target=launch_analytics_thread, args=(data,)).start()
 
 
+def launched_telemetry(blocks: gradio.Blocks, data: Dict[str, Any]) -> None:
+    from gradio.blocks import BlockContext
```

Comment:
Why import inside the function? Better to keep all imports at the top level if possible